media-libs/bpg/bpg-0.9.8.recipe

```diff
@@ -22,7 +22,9 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
-	lib:libpng$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
```

Comment:
What part of this requires SDL? Can we break that out into a subpackage?

Reply:
The image viewer bpgview (which isn't installed by default), I could check to see if I can disable that one ...

Reply:
Changed the recipe a bit so it doesn't build the viewer

Reply:
Checked with installing the binary bpgview, can't split the cmd's into separate packages?